### ISSUE TITLE
Headless feature Glutin not available

### ISSUE THREAD

**Issue by coenert** [gfx-rs/gfx#1523]
In the cargo.toml of the gfx_window_glutin there is a feature defined called 'headless', but i can't find it back on crates.io, even though this feature is added before the last release, i believe.

I'm i doing something wrong or is this something to be fixed?

**Comment by vitvakatu** [comment]
This feature is used for CI tests to ckeck code without creating window (that's impossible)

**Comment by msiglreith** [comment]
> even though this feature is added before the last release

I can't find it on the `0.16` branch, so it shouldn't be included in the crates.io release. We probably missed adding it for this branch.

**Comment by kvark** [comment]
@coenert care to provide a PR for v0.6 branch, so that we can publish an updated gfx_window_glutin?

**Comment by Kerollmops** [comment]
Just to know, where does the `gfx_window_glutin` have been "moved" ?
Here is my [master branch](https://github.com/gfx-rs/gfx/commit/05e34beabc617ce7eda7117c6686ba0a2db5b337).

**Comment by msiglreith** [comment]
It's in the `pre-ll` branch: https://github.com/gfx-rs/gfx/tree/pre-ll
We are currently rewriting the core on master, the "old" gfx code lives in the pre-ll branch.

**Comment by Kerollmops** [comment]
Did you means that something will change about `gfx_window_glutin` ? Will it be rewritten ?

**Comment by msiglreith** [comment]
Yes. The window crates have been merged with the backend crates on master, including a rewrite to align the window creation process of the different backends (Window -> Surface -> Swapchain). But this will still take some time. We currently actively develop the rewrite and maintain the `pre-ll` (v0.16) branch.
